### DISCUSSION TITLE
anybody have any working gimkit and blooket hacks and how to.

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1379] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
??

**Comment by OPcoolman** [comment]
lol

**Comment by S-PScripts** [comment]
https://github.com/S-PScripts/chromebook-utilities/blob/main/Cheats/Blooket%20Cheats
https://github.com/S-PScripts/chromebook-utilities/blob/main/Cheats/Gimkit%20Cheats

you will need to have bookmarklets (javascript: in bookmarks) available to run these

**Reply by ghost** [reply]
Thanks!
